Bottom: 86715587f396afc7b4fd5221139b13723eb6a595
Top:    0bfe9c8e4a3f849b95898baa969fd4e31609df81
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:31:48 +0000

Mark segments as not round before falling back to line-line intersection

This avoids assert failures with debugging enabeled


---

diff --git a/src/polygon1.c b/src/polygon1.c
index fd3b284..fb25283 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1320,6 +1320,8 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
+      s1->v->is_round = false;
+      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
